narwhals/dataframe.py

```diff
@@ -330,10 +330,22 @@ def unpivot(
         variable_name: str | None,
         value_name: str | None,
     ) -> Self:
+        on_ = [on] if isinstance(on, str) else on
+        index_ = (
+            [index]
+            if isinstance(index, str)
+            else index
+            if isinstance(index, list)
+            else []
+        )
+
+        variable_name = variable_name if variable_name is not None else "variable"
+        value_name = value_name if value_name is not None else "value"
```

Comment:
🤔 I don't understand why the signature isn't `variable_name: str = "variable"` in Polars, am I missing something?

Reply:
Good point! Not sure why they do it like that